internal/receiver/databricksreceiver/documentation.md

```diff
@@ -8,18 +8,166 @@ These are the metrics available for this scraper.
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.spark.util.executortimelogginglistener.mean** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.spark.util.usagelogginglistener.mean** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.sql.advice.advisorlistener.mean** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.sql.debugger.querywatchdoglistener.mean** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.sql.execution.ui.iocachelistener.mean** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.sql.io.caching.repeatedreadsestimator.mean** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.queue.appstatus.listenerprocessingtime.mean** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.queue.executormanagement.listenerprocessingtime.mean** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.queue.shared.listenerprocessingtime.mean** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.queue.streams.listenerprocessingtime.mean** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>app_id</li> </ul> |
+| **databricks.tasks.run.duration** | The execution duration in milliseconds per completed task | ms | Gauge(Int) | <ul> <li>job_id</li> <li>task_id</li> </ul> |
+| **databricks.tasks.schedule.status** | A snapshot of the pause/run status per task taken at each scrape 0=PAUSED, 1=UNPAUSED, 2=NOT_SCHEDULED | {status} | Gauge(Int) | <ul> <li>job_id</li> <li>task_id</li> <li>task_type</li> </ul> |
+
+**Highlighted metrics** are emitted by default. Other metrics are optional and not emitted by default.
+Any metric can be enabled or disabled with the following scraper configuration:
+
+```yaml
+metrics:
+  <metric_name>:
+    enabled: <true|false>
+```
+
+## Resource attributes
+
+| Name | Description | Type |
+| ---- | ----------- | ---- |
+| databricks.instance.name | The name of the Databricks instance as defined by the value of the "instance_name" field in the config | Str |
+
+## Metric attributes
+
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| app_id | n/a |  |
+| cluster_id | n/a |  |
+| executor_id | n/a |  |
+| job_id | The numeric ID of the Databricks job |  |
+| spark_job_id | n/a |  |
```

Comment:
can any of those be resource attributes? Description for those would be very helpful